bigframes/series.py

```diff
@@ -323,7 +323,7 @@ def to_pandas(
         sampling_method: Optional[str] = None,
         random_state: Optional[int] = None,
         *,
-        ordered: bool = True,
+        ordered: Optional[bool] = None,
```

Comment:
I might not fully follow the conversation. How to handle the breaking changes usually?

Reply:
I guess I don't see it as a breaking change necessarily, since all existing code should work the same `to_pandas()` without ordered parameter will still be ordered by default for all existing code.